replicate-to-sql-with-generic-events/lib/postgres.js

```diff
@@ -0,0 +1,52 @@
+const { Client } = require('pg');
+
+const pgConfig = {
+  user: 'my_user',
+  database: 'nyc_open_data',
+  password: 'password',
+  port: 5432
+};
+
+const pgConfigLocal = Object.assign({ host: 'localhost' }, pgConfig);
+const pgConfigDocker = Object.assign({ host: 'postgresql' }, pgConfig);
+
+class PostgresWrapper {
+  constructor(config) {
+    this.config = config;
+    this.client = new Client(config);
+  }
+
+  connect() {
+    return this.client.connect();
+  }
+
+  end() {
+    return this.client.end();
+  }
+
+  formatPlaceholders(values) {
+    return values.map((_, i) => `$${i + 1}`).join(',');
+  }
+
+  async insert(data) {
+    const [params, values] = Object.entries(data);
```

Comment:
Reading this function's code, I wonder... are you sure this is what you want to do?

```js
const foo = {foo: 1, bar: 2};
const [k, v] = Object.entries(foo);

console.dir(k);
// prints: [ 'foo', 1 ]
```

Reply:
Oh man I feel guilty on this one..

Reply:
Just wanted to add that it worked with that code somehow

Reply:
How's that possible? :thinking: